### PR TITLE
fix the bug of ktx2's texture minfilter

### PR DESCRIPTION
**Pull request by vtxf**
It only happens for ktx2 texture. cesium does not use gltf's origin minfilter.

origin:
image

fixed:
image

test gltf file:
[t2.zip](https://github.com/CesiumGS/cesium/files/7352635/t2.zip)

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -168,6 +168,8 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
   - [Mark Aubin](https://github.com/markaubin)
   - [Michael Shantzis](https://github.com/shantzis1962)
   - [Sebastian Fernandez](https://github.com/SebastianPFernandez)
+- [Beijing Xibushijie]
```
Comment (ebogo1) [inline CONTRIBUTORS.md:171]: Do you have a link to your company website by any chance? i.e. https://cesium.com/. If not, there's no need for brackets around the company name.
```suggestion
- Beijing Xibushijie
```

**Comment by ebogo1** [comment]
This should also fix https://github.com/CesiumGS/cesium/issues/9746:

On this branch: 
image

On `main`: 
image

File: CHANGES.md [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 - Added `ScreenOverlay` support to `KMLDataSource`. [#9864](https://github.com/CesiumGS/cesium/pull/9864)
 
+##### Fixes :wrench:
+
+- fix the bug of ktx2 texture minfilter.
```
Comment (ebogo1) [inline CHANGES.md:11]: Please also merge `main` in again and put this under a new `##### Fixes :wrench:` section for `### 1.88 - 2021-12-1`. Thanks!

```suggestion
- Fixed a bug with .ktx2 textures having an incorrect minification filter. [#9876](https://github.com/CesiumGS/cesium/pull/9876/)
```

**Review by ebogo1 — COMMENTED** [review]
Thanks @vtxf! This change looks good to me, and this should be good to merge to `main`. Since we missed the 1.87 release on this one (sorry about that), could you please merge `main` into your branch and update the CHANGES.md entry to put the fix into a new section for the 1.88 release? 

I had a couple minor suggestions and this should be ready to go.

**Comment by vtxf** [comment]
> Thanks @vtxf! This change looks good to me, and this should be good to merge to `main`. Since we missed the 1.87 release on this one (sorry about that), could you please merge `main` into your branch and update the CHANGES.md entry to put the fix into a new section for the 1.88 release?
> 
> I had a couple minor suggestions and this should be ready to go.

Thanks for your suggestions and I have updated the branch.

**Comment by ebogo1** [comment]
Thanks again @vtxf! I pushed a small commit to update CHANGES.md after a mini release we did today.

**Pull request merged**